metadata/make_merged_CDE_table.py

```diff
@@ -0,0 +1,50 @@
+#!/bin/env python
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, Gregory Poore.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# ----------------------------------------------------------------------------
+
+import numpy as np
+import pandas as pd
+import subprocess
+import glob
+
+## Call Bash script to transpose data
+print "Transposing CDE file..."
+subprocess.call("./text_transpose.sh", shell=True)
```

Comment:
Best to avoid hardcoded paths. Could the path to the metadata directory be an argument to ```make_merged_CDE_table.py```?